vertica/datadog_checks/vertica/utils.py

```diff
@@ -0,0 +1,26 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base.constants import ServiceCheck
+
+# https://www.vertica.com/docs/9.2.x/HTML/Content/Resources/Images/Node_States_531x851.png
+#
+# UP is OK, anything on a possible path to UP is WARNING (except DOWN), otherwise CRITICAL
+NODE_STATES = {
```

Comment:
You may want to look at IBM_MQ custom mapping feature

Reply:
Do you mean this? https://github.com/DataDog/integrations-core/blob/bb4193689bb015e6855ef252293729f0e23ab506/ibm_mq/datadog_checks/ibm_mq/config.py#L28-L39

Reply:
yes, the user can provide their own mapping